_devExtensions/api.js

```diff
@@ -6,6 +12,87 @@ const getIp = {
   }
 }
 
+const removeOldestPassword = (db, userId) => {
+  const query = `DELETE FROM ${tableName} WHERE userId = ? ORDER BY createdAt LIMIT 1`
```

Comment:
@TahaKhanAbdalli - can we structure the query to delete all except the most recent 5, just in case?